content/300-accelerate/650-troubleshoot.mdx

```diff
@@ -110,9 +110,11 @@ This error indicates that Prisma Accelerate cannot establish a connection to you
 
 #### Database Not Publicly accessible
 
-Prisma Accelerate currently requires the database to be publicly accessible. If your database is behind a VPC, or access is restricted to specific IP addresses, this error may occur.
+If your database is behind a VPC, or access is restricted to specific IP addresses, this error may occur.
 
-**Suggested solution:** Ensure your database is publicly accessible, if not, allow public access. Please note this is only an interim requirement, Static IPs are coming soon and will alleviate the need for complete public access if that’s not desirable.
+**Suggested solution:** You'll have to enable static IP within your Accelerate configuration in your project environment. After you opt-in to use Accelerate with static IP enabled, add the IP address you receive in the database firewall IP allowlist or permit it access to your private network. This will ensure that Accelerate can securely access your database.
+
+> ℹ️ To enable static IP support for Accelerate within your existing or new project environment, ensure you're on our **Pro** or **Business** plan. Refer to our [pricing page](https://www.prisma.io/pricing) for more information. To learn more about static IP see this [FAQ](/accelerate/faq#when-do-i-have-to-enable-static-ip-for-prisma-accelerate).
```

Comment:
Do we really need this callout everywhere? Is there a first-class location for Static IP in the docs where we can just link mentions of it, so people can click there and see these limitations?

Reply:
We can keep the [FAQ](https://docs-git-static-ip-prisma.vercel.app/accelerate/faq#when-do-i-have-to-enable-static-ip-for-prisma-accelerate) as a first class location where I can link mentions to it.